signer/core/apitypes/types_test.go

```diff
@@ -39,3 +48,94 @@ func TestIsPrimitive(t *testing.T) {
 		}
 	}
 }
+
+func TestTxArgs(t *testing.T) {
+	for i, tc := range []struct {
+		data     []byte
+		want     common.Hash
+		wantType uint8
+	}{
+		{
+			data:     []byte(`{"from":"0x1b442286e32ddcaa6e2570ce9ed85f4b4fc87425","accessList":[],"blobVersionedHashes":["0x010657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014"],"chainId":"0x7","gas":"0x124f8","gasPrice":"0x693d4ca8","input":"0x","maxFeePerBlobGas":"0x3b9aca00","maxFeePerGas":"0x6fc23ac00","maxPriorityFeePerGas":"0x3b9aca00","nonce":"0x0","r":"0x2a922afc784d07e98012da29f2f37cae1f73eda78aa8805d3df6ee5dbb41ec1","s":"0x4f1f75ae6bcdf4970b4f305da1a15d8c5ddb21f555444beab77c9af2baab14","to":"0x1b442286e32ddcaa6e2570ce9ed85f4b4fc87425","type":"0x1","v":"0x0","value":"0x0","yParity":"0x0"}`),
```

Comment:
Hm `"type": "0x1"` but expect blob tx type? didnt look too deep tho

Reply:
See the comment below: 
```
// on input, we don't read the type, but infer the type from the arguments present
```
So the test-json contain bogus `type`, which geth ignores. And the test expects geth to ignore it

Reply:
I was wondering why it's not failing on invalid parameters. But yeah TransactionArgs (and SendTxArgs) don't have the `type` field and validate it. myeah.